### ISSUE TITLE
Filters - 2 columns

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-policies-prototype#186]
I'm getting significant wrapping, even on my external monitor when I zoom down to 100%. Makes it harder to process the filter categories and options.

I suggest two columns:
1. WCAG Version Used and Scope of Policy
2. Type of Policy

<img width="673" alt="125" src="https://cloud.githubusercontent.com/assets/6991814/25910815/47cf20c4-3577-11e7-99d4-31d792a59f36.png">
<img width="649" alt="100" src="https://cloud.githubusercontent.com/assets/6991814/25910834/539d5434-3577-11e7-8543-1b4dbe579589.png">

**Comment by yatil** [comment]
This is not responsive yet. (And with the table it never can be. I’ll look into it.)

**Comment by slhenry** [comment]
hum -- maybe I'm not clear. I think by default there should be only 2 columns.

**Comment by yatil** [comment]
understood.

**Comment by yatil** [comment]
people just said wasted space which is why we compressed it.
probably no issue with expand/collapse.

**Comment by slhenry** [comment]
agree wasted space when it was just one column.

**Comment by yatil** [comment]
we’re constrained with the html due to how the script works. older browsers have a three column layout now, modern browsers get this enhanced version: https://dsc.cloud/yatil/Screen-Shot-2017-05-10-22-59-01

**Comment by slhenry** [comment]
+1 !

**Comment by slhenry** [comment]
(if easy, could add space above Scope so it's a little more separated from the one above it. :-)

**Comment by iamjolly** [comment]
Something like this? 
![screenshot 2017-05-10 15 16 35](https://cloud.githubusercontent.com/assets/1093423/25921728/c66ae9f6-3593-11e7-8172-2c60185726f5.png)

**Comment by slhenry** [comment]
+1 to @iamjolly  -- I think that improves usability

**Comment by yatil** [comment]
I will align the scope to the bottom of the grid – I meant to do that. (It would be good if we would not work on the same issues at the same time :-)

**Comment by iamjolly** [comment]
Thanks @yatil - Your solution is the better one, I think.
